test/plugins.js

```diff
@@ -1,6 +1,7 @@
 import { join } from 'node:path';
+import sh from 'node:child_process';
```

Comment:
Let's use the same name for this default import everywhere.

Reply:
Fixed